quickstart/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,22 @@
+####
```

Comment:
It's a little odd that we just have these Dockerfiles hanging out here with no obvious context; we talk about running the app locally in the README but there's no mention of running in a container.  Maybe we should at least mention these there?

Reply:
The readme is in the dockerfiles:
https://github.com/datastax/cassandra-quarkus/pull/8/files/51bdfadffad1fda23ab550529de769cc4f20af10#diff-5b11c784af46052926eaa14bfec26b15R2-R16

Reply:
Yeah, I saw that @tomekl007 .... I just meant that it seemed odd that the main README talks about running all the apps locally (with no mention of Docker) and then there are separate Dockerfiles provided to do the same thing.  It's not a huge thing it just seemed a little jarring.

Reply:
I am following the same pattern that is for other quick-starts. They all have those dockerfiles.